src/core/reference.cc

```diff
@@ -6,12 +6,6 @@ PythonReference::PythonReference(PythonObject *scope) : mScope(scope) {
   mCapsule = PyCapsule_New(this, nullptr, nullptr);
 }
 
-PythonReference::~PythonReference() {
```

Comment:
[F] This Unref was causing the same errors shown in nodejs/node-addon-api#998. Removing the Unref gets rid of the errors. The explanation for why we shouldn't be doing this is shown in the issue here.

I originally left the method in and empty but that led to linting errors.

Reply:
😢